src/examples/hid-keyboard/main.go

```diff
@@ -1,11 +1,19 @@
+// to override the USB Manufacturer or Product names:
+//
+// tinygo flash -target circuitplay-express -ldflags="-X main.usbManufacturer='TinyGopher Labs' -X main.usbProduct='GopherKeyboard'" examples/hid-keyboard
```

Comment:
It would be helpful if it said that the VID/PID can be changed in the same way, but with caution.

Reply:
Updated.